playwright/README.md

```diff
@@ -39,6 +39,22 @@ To run just a specific test:
 npx playwright test tests/root_password.spec.ts
 ```
 
+## Updating the Screenshots
+
+There is one test specially designed for refreshing the screenshots displayed
+in the main `README.md` file.
+
+To fully run the installation run this:
```

Comment:
np: I would replace "run" with "type" (or something similar) to not repeat the term "run".